care/security/management/commands/sync_permissions_roles.py

```diff
@@ -44,7 +44,9 @@ def handle(self, *args, **options):
                 role_obj.save()
             RoleModel.objects.filter(temp_deleted=True).delete()
             # Sync permissions to role
-            RolePermission.objects.all().update(temp_deleted=True)
```

Comment:
Permissions are all system by default. Permissions cannot be dynamically created.

Reply:
L47 would be required in the case, where a system role had a permission earlier but it has been removed from the permission.roles list

Reply:
RolePermission is the through table

Reply:
Ignore dumb me and my dumb questions 😓